lighthouse-core/gather/gatherers/trace-elements.js

```diff
@@ -16,6 +16,8 @@ const pageFunctions = require('../../lib/page-functions.js');
 const TraceProcessor = require('../../lib/tracehouse/trace-processor.js');
 const RectHelpers = require('../../lib/rect-helpers.js');
 
+/** @typedef {{nodeId: Number, score?: Number}} TraceElementData */
```

Comment:
`number`. see https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#general-types